gapis/service/memory_box/box.proto

```diff
@@ -0,0 +1,53 @@
+// Copyright (C) 2019 Google Inc.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+syntax = "proto3";
+
+import "core/data/pod/pod.proto";
+
+package memory_box;
+option java_package = "com.google.gapid.proto.service.memory_box";
+option java_outer_classname = "MemoryBox";
+option go_package = "github.com/google/gapid/gapis/service/memory_box";
+
+message Value {
+  oneof val {
+    pod.Value pod = 2;
+    Pointer pointer = 3;
+    Slice slice = 4;
+    Struct struct = 5;
+    Array array = 7;
+    Null null = 8;
```

Comment:
Is there a good reason to skip value 6 in this oneof ?

Reply:
Nope, there is not, fixed.